test/unit/browser/pages/vscode.test.ts

```diff
@@ -87,6 +119,13 @@ describe("vscode", () => {
       const actual = createBundlePath(_resolvedLangaugePackCoreLocation, bundle)
       expect(actual).toBe(expected)
     })
+    it("should return the correct path (even if _resolvedLangaugePackCoreLocation is undefined)", () => {
```

Comment:
✅ This check was added after modifying the `createBundlePath` function signature.